module/VuFind/src/VuFind/Search/UrlQueryHelper.php

```diff
@@ -76,7 +76,7 @@ class UrlQueryHelper
      * on the contents of $query to $urlParams (true) or are they already there
      * (false)?
      */
-    public function __construct(array $urlParams, AbstractQuery $query,
+    final public function __construct(array $urlParams, AbstractQuery $query,
```

Comment:
Why make this final?

Reply:
All the finals are because of using `new static()`, see https://phpstan.org/blog/solving-phpstan-error-unsafe-usage-of-new-static

another option is to define interface which defines constructor, or ignore this error.

Reply:
I'll look into it when I'm back in the office. Thanks for the explanation!